EdgeOfEmpireBot/EdgeOfEmpireBot/Service/MessageService.cs

```diff
@@ -71,25 +83,45 @@ public bool MessageIsSafe()
         /// <param name="command"></param>
         private async Task ProcessCommand(string command, ISocketMessageChannel channel)
         {
-            // Check if the command is a custom command that requires functionality, ie a roll. 
-            // If not a custom command try processing it as a simple command. 
-            // TODO: Consider the order of this once the scope of overall commands have been established. For efficency. 
-            var commandNoParam = command.Split(' ').FirstOrDefault();
-            switch (commandNoParam)
+            // Check if the command is a custom command that requires functionality, ie a roll.
+            // If not a custom command try processing it as a simple command.
+            // TODO: Consider the order of this once the scope of overall commands have been established. For efficency.
+            var commandParams = command.Split(' ');
+            var commandNoParam = commandParams[0];
```

Comment:
not a first or default now?

Reply:
Its not necessary since this achieves the same result without the overhead of the Linq Query

Reply:
I thought first default was just more radable than 0th index ahah happy either or